api/python/provisioner/inputs.py

```diff
@@ -877,3 +879,60 @@ def is_iso(self):
 @attr.s(auto_attribs=True)
 class SWUpgradeRepo(SWUpdateRepo):
     _param_di = param_spec['swupgrade/repo']
+    # file path to base directory for SW upgrade
+    _target_build: str = attr.ib(default=None)
```

Comment:
looks good, JFYI: you may shorten that more as:
```
_target_build: str = None
_enabled: bool = False
```
due to `@attr.s(auto_attribs=True)` [attr lib](https://www.attrs.org/en/stable/api.html) will get that right as well (search for `auto_attribs`)